backend/connector-integration/src/connectors/globalpay/transformers.rs

```diff
@@ -261,7 +262,7 @@ pub struct GlobalpayNotifications {
     pub status_url: String,
 }
 
```

Comment:
The `Clone` and `Copy` traits are added but never used. Consider if these traits are actually necessary for these enums.

If they're not required by the serialization or usage elsewhere, the simpler approach would be to keep just `Debug, Serialize`:

```suggestion
#[derive(Debug, Serialize)]
```